notebooks/1_BayesianWorkflow/3_Ins_EndtoEndWorkflow.ipynb

```diff
@@ -0,0 +1,178 @@
+   "source": [
+    "## The Bayesian Landscape\n",
+    "\n",
+    "### Inference\n",
+    "In our planet model we were trying to *infer* the proportion of water on a planet with only a few observations of the planet using Bayes Rule. More specifically we were making estimates by sampling the posterior distribution using a **specific** inference method called Markov Chain Monte Carlo. \n",
+    "\n",
+    "This is important because there are [many different inference methods](http://canyon289.github.io/pages/BayesianGlossary.html). A short list includes\n",
+    "* Conjugate Priors\n",
+    "* Grid Search\n",
+    "* Variational Inference\n",
+    "* Markov Chain Monte Carlo\n",
+    "\n",
+    "At time of writing however Markov Chain Monte Carlo is one of the most widely used and will be the focus of the tutorial.\n",
+    "\n",
+    "### Markov Chain Monte Carlo methods to estimate posterior distributions\n",
+    "To add to the mix there are numerous Markov Chain Monte Carlo samplers. A short list of common ones are\n",
+    "* Metropolis Hasting\n",
+    "* Hamiltonian Monte Carlo (HMC)\n",
+    "* No U Turn Samplers (NUTS)\n",
+    "\n",
+    "### Probabilistic Programming Languages\n",
+    "And to make things more confusing there are multiple programatic implementations of Bayesian Inference methods of all types in multiple languages\n",
+    "* PyMC3 in Python\n",
+    "* PyMC4 in Python\n",
+    "* pystan in Python\n",
+    "* emcee in Python\n",
+    "* Edward in Python\n",
+    "* Pyro in Python\n",
+    "* cmdstan \n",
+    "* [rainier](https://github.com/stripe/rainier) in scala\n",
+    "* [Turing](https://github.com/TuringLang/Turing.jl) in julia\n",
+    "* [Anglican](https://probprog.github.io/anglican/index.html) in clojure\n",
+    "* [Infer.net](https://dotnet.github.io/infer/)\n",
+    "* [Webppl](http://webppl.org/) in javascript\n",
+    "* [Greta](https://greta-stats.org/) in r\n",
+    "\n",
+    "### What we will focus on\n",
+    "* Python language\n",
+    "* MCMC Inference Methods\n",
+    "* Pystan and PyMC3\n",
+    "\n",
+    "To be very specific the full title of this tutorial would be *Introduction to Bayesian Model Evaluation, Visualization, and Comparison Using Arviz and Markov Chain Monte Carlo inference methods, primarily  NUTS, with PyMC3 or pystan*"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## The End to End Bayesian Workflow\n",
+    "**The aim of statistics and machine learning should not be to just fit models, but to create defendable and reproducible analysis and results**\n",
+    "\n",
+    "This tutorial focuses on how to use ArviZ to do just that using all steps of this workflow.\n",
+    "\n",
+    "![BayesianWorkflow](../../lessonplans/img/BayesianWorkflow.jpg)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## ArviZ unifies of Bayesian Model evaluation and criticism across inference libraries\n",
```

Comment:
Following our paper I would say ArviZ is about "Exploratory analysis of Bayesian models" and that encompass 

• Diagnoses of the quality of the inference
• Model criticism, including evaluations of both model assumptions and model predictions
• Comparison of models, including model selection or model averaging
• Preparation of the results for a particular audience